pkg/private/archive.py

```diff
@@ -317,15 +369,16 @@ def add_tar(self,
       name_filter: filter out file by names. If not none, this method will be
           called for each file to add, given the name and should return true if
           the file is to be added to the final tar and false otherwise.
-      root: place all non-absolute content under given root directory, if not
-          None.
+      root: place all content under given root directory, if not None.
```

Comment:
Could I put `root=.`, `root=../` or `root=./foo`?

Reply:
That will become unsupported in the followup CL that fixes pkg_tar to not have it.